here-naksha-lib-model/src/commonMain/kotlin/naksha/model/XyzProxy.kt

```diff
@@ -192,4 +194,18 @@ class XyzProxy : ObjectProxy() {
      * This field is populated only by **Naksha**. Any values provided by the user are overwritten.
      */
     var geoGrid: Int by INT
+
+    var deltaProxy: NakshaDeltaProxy? by DELTA_PROXY
+
+    /**
+     * References to MOM objects.
+     */
+    var references: XyzReferencesProxy? by REFERENCES
+
+    fun useDeltaNamespace(): NakshaDeltaProxy {
+        if (deltaProxy == null) {
+            deltaProxy = NakshaDeltaProxy()
+        }
+        return deltaProxy!!
```

Comment:
To prevent usage of !!, simply copy variable to stack, this is even faster, because it avoids two times reading the value from the underlying hash-map, so:
```
var deltaProxy = this.deltaProxy
if (deltaProxy == null) {
  deltaProxy = NakshaDeltaProxy()
  this.deltaProxy = deltaProxy
}
return deltaProxy
```
I know that this is slightly more to type, but at runtime it is faster, and avoids potential concurrency issues.